hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -424,4 +424,6 @@ private OMConfigKeys() {
       = "ozone.om.enable.ofs.shared.tmp.dir";
   public static final boolean OZONE_OM_ENABLE_OFS_SHARED_TMP_DIR_DEFAULT
       = false;
+  public static final String OZONE_OM_SNAPSHOT_DIFF_DB_DIR
+      = "ozone.om.snapshot.diff.db.dirs";
```

Comment:
Remove `s` at the end?

```suggestion
      = "ozone.om.snapshot.diff.db.dir";
```